src/cli/api/trackingplans.ts

```diff
@@ -131,9 +135,9 @@ export function parseTrackingPlanName(
 export function toTrackingPlanURL(trackingPlan: RudderAPI.TrackingPlan): string {
   if (!trackingPlan.creationType) {
     const { id } = parseTrackingPlanName(trackingPlan.name);
-    return `https://api.rudderstack.com/trackingplans/${id}`;
+    return `https://app.rudderstack.com/trackingplans/${id}`;
   }
-  return `https://api.rudderstack.com/tracking-plans/${trackingPlan.id}`;
+  return `https://app.rudderstack.com/tracking-plans/${trackingPlan.id}`;
```

Comment:
`https://app.dev.rudderlabs.com/trackingPlans/tp_2jeqGKhl`

this seems to be url in UI